packages/wp-story-editor/src/components/documentPane/status/status.js

```diff
@@ -38,7 +37,7 @@ const InputRow = styled(Row)`
   margin-left: 34px;
 `;
 
-function StatusPanel({ nameOverride }) {
+function StatusPanel({ nameOverride, popupZIndex }) {
```

Comment:
I decided to pass `popupZIndex` as a prop so that when we render `StatusPanel` in the sidebar Document Pane, it the dropdown doesn't have to have the same zIndex as is needed when it is rendered in the modal. If we had let them be the same (ie 11 is need on the modal) then tooltips would get placed behind the dropdown.